daemons/based/based_callbacks.c

```diff
@@ -1277,7 +1289,11 @@ cib_process_command(xmlNode * request, xmlNode ** reply, xmlNode ** cib_diff, gb
         cib__set_call_options(call_options, "call", cib_force_diff);
         crm_trace("Global update detected");
 
-        CRM_CHECK(call_type == 3 || call_type == 4, crm_err("Call type: %d", call_type);
+        CRM_CHECK(pcmk__str_any_of(operation->name,
+                                   PCMK__CIB_REQUEST_APPLY_PATCH,
+                                   PCMK__CIB_REQUEST_REPLACE,
+                                   NULL),
+                  crm_err("Call type: %s", operation->name);
                   crm_log_xml_err(request, "bad op"));
```

Comment:
FYI we don't need the crm_err() or crm_log_xml_err() here; the CHECK logs failures (showing the condition tested), and this isn't a check we really care much about. But no need to change since it will all go away at some point anyway

Reply:
No need to *not* change it either, since there are other edits to make :)